docs/how-to-guides/control-the-case-of-identifiers.md

```diff
@@ -0,0 +1,103 @@
+# How to control the case of identifiers
+
+## Context
+
+You have identifiers, like row names, that will be used outside of
+ksqlDB. You want to control the exact casing (capitalization) of how
+they are represented to make them consumable by downstream
+programs. Because ksqlDB uppercases all identifiers by default, you
+need to use backticks to preserve the desired casing.
+
+## In action
+
+```sql
+CREATE STREAM `s1` (
+    `a` VARCHAR KEY,
+    `B` INT,
+    `c` VARCHAR
+) WITH (
+    kafka_topic = 's1',
+    partitions = 1,
+    value_format = 'avro'
+);
+```
+
+## Backticks
+
+Begin by telling ksqlDB to start all queries from the earliest point
+in each topic.
+
+```sql
+SET 'auto.offset.reset' = 'earliest';
+```
+
+Declare a new stream named `s2`. In this example, you override
+ksqlDB's default behavior to uppercase all identifiers. Use backticks
+to control the casing of the stream name and two of the column names,
+`a` and `B`. `c` receives the default behavior of uppercasing.
+
+```sql
+CREATE STREAM `s2` (
+    `a` VARCHAR KEY,
+    `B` INT,
+    c VARCHAR
+) WITH (
+    kafka_topic = 's2',
+    partitions = 1,
+    value_format = 'avro'
+);
+```
+
+Insert some rows into `s2`. Notice how you need to use backticks each
+time to reference a field that doesn't have the default casing:
```

Comment:
unless it's all uppercase 😂 you could reference `B` without quotes. might be good to have an example with mixed case (`` `Dog` ``)

Reply:
also if you have `` `B` `` and `` `b` `` as two different fields, `b` would resolve to `` `B` `` and `` `b` `` would resolve to `` `b` `` (correctly)

Reply:
I think I covered this without making it too confusing. See what you think.